ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -1544,7 +1579,7 @@ def test_paddle_angle(
     helpers.test_frontend_method(
         init_input_dtypes=input_dtype,
         init_all_as_kwargs_np={
-            "data": x[0],
+            "value": x[0],
```

Comment:
I think this change was made by mistake. Could you kindly revert this 🙂

Reply:
Hello @YushaArif99 , 

I'm not clear on this, the changes were on different lines.
I think the lines were making the test fail and I had to update it